embed-video.md

```diff
@@ -19,3 +19,58 @@
 - `muted`     : Sets the initial muted state. `true` or `false`. Defaults to `false`.
 - `autoplay`  : Automatically starts playing without the user clicking play. `true` or `false`. Defaults to `true`.
 - `time`      : Start playback at the given timestamp for videos ONLY. Must be in the format `1h2m3s` specifying hours, minutes, and seconds respectively. Defaults to the start of the video.
+
+###Video and Channel Metadata
+For video and channel metadata refer to the [Twitch API (Kraken)] (https://github.com/justintv/Twitch-API).
+For convenience, some available metadata is listed below for convenience. All are available by [`GET/videos/:id`] (https://github.com/justintv/Twitch-API/blob/master/v3_resources/videos.md#get-videosid). 
+- Video length
+- Video title
+- Video description
+- Video viewcounts
+
+
+##Interactive Embed Example (Not Available Yet)
+    <div id="video-playback"></div>
+	<script type="text/javascript">
+		var options = {
+			width: 854,
+			height: 480,
+			channel: "{CHANNEL}", //video: "{VIDEO_ID}",
+		};
+
+		var player = new Twitch.embed.Player("video-playback", options);
```

Comment:
Is this really the namespace we're going with? Also should probably add a comment that `"video-playback"` matches the ID of the `div`.